### PR TITLE
Proposed changes to TaskbarControllerTest

### PR DESCRIPTION
**Pull request by farmerbb**
@utzcoz I went ahead and made some changes based off of the comments I left in #114.  Let me know what your thoughts are on this.

### CODE REVIEW

**Comment by utzcoz** [comment]
@farmerbb It's more cleaner. The only thing we should consider is whether the `VisibleForTesting` can be accessed when building with `make` based on current dependencies on `Android.mk`.

**Comment by utzcoz** [comment]
@farmerbb  I can help to test it with `AOSP`.

File: app/src/test/java/com/farmerbb/taskbar/ui/TaskbarControllerTest.java [hunk 1]
```diff
@@ -182,32 +173,17 @@ public void testGetTaskbarLayoutId() {
     }
 
     private int callGetTaskbarLayoutId(String taskbarPosition) {
-        return callInstanceMethod(
-                uiController,
-                "getTaskbarLayoutId",
-                from(String.class, taskbarPosition)
-        );
+        uiController.getTaskbarLayoutId(taskbarPosition);
```
Comment (utzcoz) [inline app/src/test/java/com/farmerbb/taskbar/ui/TaskbarControllerTest.java:176]: return?

File: app/src/test/java/com/farmerbb/taskbar/ui/TaskbarControllerTest.java [hunk 2]
```diff
@@ -182,32 +173,17 @@ public void testGetTaskbarLayoutId() {
...
     }
 
     private int callGetTaskbarGravity(String taskbarPosition) {
-        return callInstanceMethod(
-                uiController,
-                "getTaskbarGravity",
-                from(String.class, taskbarPosition)
-        );
+        uiController.getTaskbarGravity(taskbarPosition);
```
Comment (utzcoz) [inline app/src/test/java/com/farmerbb/taskbar/ui/TaskbarControllerTest.java:180]: return?

**Comment by utzcoz** [comment]
@farmerbb, I add `Taskbar` with this pr to `packages/apps`, and use `make Taskbar` to build it on `AOSP` 9.0, the building succeed. And then I run `emulator` with `Taskbar`, it works fine.

File: app/src/test/java/com/farmerbb/taskbar/ui/TaskbarControllerTest.java (hunk 2 above)
Comment (farmerbb) [reply app/src/test/java/com/farmerbb/taskbar/ui/TaskbarControllerTest.java]: Sorry about that, I did this late last night where I live and kinda rushed this.

**Comment by farmerbb** [comment]
Hmm, coverage goes down slightly with this PR.

**Comment by farmerbb** [comment]
I added a stub UIHost so that TaskbarControllerTest can call the `onCreateHost()` and `onDestroyHost()` lifecycle methods, which gets the code coverage back up past where it was before.  `MockUIHost` can then be extended in the future if/when tests are added to verify that UIController views are added or removed.

**Pull request merged**